README.md

```diff
@@ -1 +1,151 @@
-# `aelf-react`
\ No newline at end of file
+# `aelf-react`
+
+- [`aelf-react`](#aelf-react)
+  - [Overview](#overview)
+  - [Install](#install)
+  - [`aelf-react@core` API Reference](#aelf-reactcore-api-reference)
+    - [AElfReactProvider](#aelfreactprovider)
+      - [Props](#props)
+      - [Example](#example)
+      - [Multi node](#multi-node)
+    - [useAElfReact](#useaelfreact)
+      - [Example](#example-1)
+    - [Notice](#notice)
+    - [Example of use](#example-of-use)
+
+## Overview
+
+At a high level, `aelf-react` is a state machine which ensures that certain key pieces of data (the user's current account, for example) relevant to your dApp are kept up-to-date. To this end, `aelf-react` uses [Context](https://reactjs.org/docs/context.html) to efficiently store this data, and inject it wherever you need it in your application.
+
+The data conforms to the following interface:
+
+```typescript
+interface AElfContextType = {
+  accountName?: string;
+  chainId?: string;
+  account?: AElfAddress;
+  defaultAElfBridge?: AElfDappBridge;
+  aelfBridges?: { [key: string]: AElfDappBridge };
+  isActive: boolean;
```

Comment:
Put isActive, activate, deactivate together.
Add more comments.
ex:
// status of xxx
isActive: boolean;
// xxx
activate : xxx